TEveEventDisplay/fcl/prolog.fcl

```diff
@@ -38,23 +38,25 @@ TEveEventDisplayBase : {
       addMCTraj				          : true 
 
     }
-  particles :   [11,13,2212,2112,211,22,212]
+    gdmlname :    "Offline/TEveEventDisplay/src/mu2e.gdml"
+    particles :   [11,13,2212,2112,211,22,212]
   }  
   
 TEveEventDisplayUpstream : {
     module_type : TEveEventDisplay
     showCRV : false
     showDSOnly : false
+    showInsidePS : true
     isMCOnly : true
     accumulate : false
     filler : {
       ComboHitCollection 		      : NULL
-      TimeClusterCollection                  : NULL
+      TimeClusterCollection       : NULL
       CosmicTrackSeedCollection 	: NULL
       CrvRecoPulseCollection		  : NULL
       CaloClusterCollection		    : NULL
-      CaloHitCollection    : NULL
-      MCTrajectoryCollection      : "BeamFilter" #"g4run"
+      CaloHitCollection           : NULL
+      MCTrajectoryCollection      : "g4run" #"BeamFilter" 
```

Comment:
What's the intention here?  Is this something that end users might normally edit?  If so I would like to change the design so that end users only make changes using overrides in their top level fcl.    Perhaps it would make more sense to define two configurations, by defining the second as a delta on the first.  Then the thing you edit is the name of the configuration in your top level fcl file.

Reply:
No the user would not edit the prolog. The driver fcls are already in the CallerExampleFcls directory

Reply:
If that's the case, why do you need to retain commented out code?